src/components/DatePicker/index.js

```diff
@@ -11,7 +11,7 @@ const PFDatePicker = props => {
   const [to, setTo] = useState('');
 
   const toValidator = date =>
-    isValidDate(from) && date >= from ? '' : 'To date must be less than from date';
+    isValidDate(from) && date >= from ? '' : '"From" date must be less than or equal to "To" date';
```

Comment:
This phrasing sounds a bit odd; you're validating the "to" date but the sentence focuses on the "From" date. Instead, it would seem more clear to say that the "to" date must be greater than or equal to the "from" date.

However, the statement isn't "incorrect", it just sounds odd, so I'll approve and let you decide whether to make another change. 😁